Cargo.toml

```diff
@@ -171,6 +171,7 @@ members = [
 default-members = [
     "clients/native",
     "clients/socks5",
+    "nym-authenticator-client",
```

Comment:
Do we need `Nym-registration-client` and `nym-ip-packet-client` here too?
(Yes I forgot about them last time)